src/models/__tests__/VersionedTemplate.spec.ts

```diff
@@ -1,6 +1,22 @@
 import casual from 'casual';
-import { Visibility } from "../Template";
+import { TemplateVisibility } from "../Template";
```

Comment:
Tests are missing for "findByTemplateId", "bestPractice", "search" and "findPublishedTemplateById".